roles/openshift_manage_node/tasks/main.yml

```diff
@@ -24,7 +24,9 @@
 
 - name: Set nodename for bootstrapped masters
   set_fact:
-    openshift:
+    openshift: {{ openshift | combine(l_openshift_node_update) }}
```

Comment:
This is missing quotes around the curly braces.

Reply:
Good catch, thanks. Fixed